generate_dlc_image_release_information.py

```diff
@@ -0,0 +1,127 @@
+import argparse
```

Comment:
is this the right place for this file? does it also belong in the release directory?

Reply:
This can be addressed in a separate PR as it might need non-trivial refactoring. The original intent was to have the buildspec execute a script directly from root folder.